js/exporter/jspdf/export.js

```diff
@@ -0,0 +1,178 @@
+        }
+        if(!isDefined(fullOptions.autoTableOptions)) {
+            fullOptions.autoTableOptions = this.getDefaultAutoTableOptions();
+        } else {
+            if(!isObject(fullOptions.autoTableOptions)) {
+                throw Error('The "autoTableOptions" option must be of object type.');
+            }
+            fullOptions.autoTableOptions = extend(true, {}, this.getDefaultAutoTableOptions(), fullOptions.autoTableOptions);
+        }
+
+        return fullOptions;
+    },
+
+    getDefaultAutoTableOptions: function() {
+        return {
+            theme: 'plain',
+            tableLineColor: 149,
+            tableLineWidth: 0.1,
+            styles: {
+                textColor: 51,
+                lineColor: 149,
+                lineWidth: 0
+            },
+            columnStyles: {},
+            headStyles: {
+                fontStyle: 'normal',
+                textColor: 149,
+                lineWidth: 0.1
+            },
+            bodyStyles: {
+                lineWidth: 0.1
+            },
+            head: [],
+            body: []
+        };
+    },
+
+    export: function(options) {
+        const {
+            jsPDFDocument,
+            autoTableOptions,
+            component,
+            keepColumnWidths,
+            selectedRowsOnly
+        } = options;
+
+        const tableWidth = component.option('width');
+        const dataProvider = component.getDataProvider(selectedRowsOnly);
+
+        return new Promise((resolve) => {
+            dataProvider.ready().done(() => {
+                const columns = dataProvider.getColumns();
+                const styles = dataProvider.getStyles();
+                const dataRowsCount = dataProvider.getRowsCount();
+
+                if(tableWidth) {
+                    autoTableOptions.tableWidth = this.convertUnitToPoints(tableWidth, 'px');
+                }
+
+                this.setColumnWidths(autoTableOptions, columns, keepColumnWidths);
+
+                for(let rowIndex = 0; rowIndex < dataRowsCount; rowIndex++) {
+
+                    let rowType = null;
+                    const row = [];
+
+                    for(let cellIndex = 0; cellIndex < columns.length; cellIndex++) {
+
+                        const cellData = dataProvider.getCellData(rowIndex, cellIndex, true);
+                        const cellStyle = styles[dataProvider.getStyleId(rowIndex, cellIndex)];
+                        const gridCell = cellData.cellSourceData;
+                        const pdfCell = {
+                            content: cellData.value,
+                            styles: {}
+                        };
+
+                        this.assignCellStyle(pdfCell, gridCell, columns[cellIndex], cellStyle);
+
+                        rowType = rowType || gridCell.rowType;
+                        if(rowType === 'header') {
+                            if(pdfCell.content !== '') { row.push(pdfCell); }
```

Comment:
else ?
Or new entry into TODO check list?

Reply:
Yes, it is wrong to check the contents of the header cell. I will remove this condition.
In the future, we will implement the header logic for BAND here.